README.md

```diff
@@ -108,11 +108,68 @@ export MON_AGENT_COLLECTOR_VERSION=ocata
 
 That file, called ```.ocata.env```, differs from [.env](./.env), as it contains ```export```
 instructions. That is mandatory approach, as sourced variables takes precedence over
-aformentioned environmental file. 
+aforementioned environmental file.
 
     Alternative approach, that achieves the same result, is to override values
     of certain variables directly in ```.env``` file.
 
+Using external OpenStack Keystone
+---------------------------------
+
+By default `monasca-docker` is using Keystone stored in separate container.
+But if you want to use external Keystone you need to overwrite some
+environment variables in specific containers.
+
+Remove `keystone` image initialization and all references to this
+image from `depends_on` from other images from `docker-compose.yml` and
+`log-pipeline.yml` files.
+
+Example configuration taking into account usage of Keystone aliases
+instead of ports when overwriting environment variables:
+
+#### `docker-compose.yml`
+
+In `agent-forwarder` and `agent-collector`:
+```yaml
+environment:
+  OS_AUTH_URL: http://192.168.10.6/identity/v3
+  OS_USERNAME: monasca-agent
+  OS_PASSWORD: password
```

Comment:
I think it's better to add "OS_PROJECT_NAME" here.

Reply:
Adding in https://github.com/monasca/monasca-docker/pull/179